src/llwe.c

```diff
@@ -106,8 +106,18 @@ static int bsave(void)
 {
 	FILE *f = fopen(filename, "w");
 	if (f == NULL) {
-		err("write");
-		return 0;
+		const char msg1[] = "Error: Failed to write to: ";
+		const char msg2[] = "Press any key to continue.";
+		char nstr[strlen(msg1) + strlen(filename) + 1];
+		snprintf(nstr, sizeof(nstr), "%s%s", msg1, filename);
+		attron(A_STANDOUT);
+		mvaddstr(LINES-2, 0, nstr);
+		mvaddstr(LINES-1, 0, msg2);
+		attroff(A_STANDOUT);
+		refresh();
+		getch();
+		err("Failed to write.");
```

Comment:
You don't need to call `err` here. If we were returning `0` then the message passed through with `err()` would eventually get printed, but since we're returning `1` there's no need.